### PR TITLE
Open insights in new tabs

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Today, there is no way to open multiple tabs of the same insight from the Insight Edit page. This PR lets you open insight tabs in new tabs using your standard Cmd/Ctrl + click/middle mouse button.

The meat of this code came from pairing session with @mariusandra—appreciate the help!

https://user-images.githubusercontent.com/13460330/144324419-2bac7c75-32a4-40a0-a4d4-4fd9277338cb.mov

Considerations

* We experimented with opening a new explorable insight whenever creating new tabs from a saved insight, but the experience didn't feel intuitive. Ultimately landed on showing the same insight because the same behavior can be expected when Cmd+clicking the same insight from the saved insights list page. 

## How did you test this code?

1. Cmd + click on any insight tab
2. Observe new tab open up with the same insight

### CODE REVIEW

File: frontend/src/scenes/experiments/experimentsLogic.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { kea } from 'kea'
 import { api } from 'lib/api.mock'
 import { teamLogic } from 'scenes/teamLogic'
 import { Experiment } from '~/types'
+import { experimentsLogicType } from './experimentsLogicType'
```
Comment (alexkim205) [inline frontend/src/scenes/experiments/experimentsLogic.tsx:5]: might as well

File: frontend/src/scenes/insights/Insight.scss [hunk 2]
```diff
@@ -15,7 +15,24 @@ $funnel_canvas_background: #fafafa;
...
-            padding-bottom: 6px;
+
+            .ant-tabs-tab-btn {
+                height: 100%;
+                width: 100%;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+
+                .tab-text {
```
Comment (alexkim205) [inline frontend/src/scenes/insights/Insight.scss:26]: Expanding inner span to make it clickable

**Review by mariusandra — COMMENTED** [review]
I think this will have a merge conflict once https://github.com/PostHog/posthog/pull/7441 lands. Otherwise the code looks good to me.

However my question is the same I raised during pairing: it still feels kind of weird to have multiple tabs open, where you're editing the same object. Mostly because if you save in one of those tabs, the other tabs won't know anything about it, and what the user does next is anybody's guess.

Thus I'd pool more opinions before ✅'ing this. CC @clarkus as well.

**Review by paolodamico — APPROVED** [review]
love this! I've had this pain multiple times where I want to open 2-3 tabs to create different insights on each one.

**Review by Twixes — DISMISSED** [review]
I'm a bit worried about the potential conflict if someone ends up trying to save one insight twice this way too, but that should be alright, let's give users this possibility.
However, there's a merge conflict that needs to be resolved – changes from #7441 need to be preserved here.

**Comment by alexkim205** [comment]
Echoing @Twixes I think it's an alright trade off to have, especially since it's already possible to have two instances of the same insight in different tabs (if you really wanted it). I'll cleanup the merge conflicts and merge this in then

**Pull request merged**